po/fr.po

```diff
@@ -131,319 +144,451 @@ msgid "Highlight focused application"
+msgstr "Longueur maximum (px) des titres d'applications (défaut: 160)"
+
+#: Settings.ui.h:31
+msgid "Use a fixed width for the application titles"
+msgstr "Utiliser une longueur fixe pour les titres d'application"
+
+#: Settings.ui.h:32
+msgid ""
+"The application titles all have the same width, even if their texts are "
+"shorter than the maximum width. The maximum width value is used as the fixed "
+"width."
+msgstr ""
+"Les titres d'application ont tous la même longueur, même si leur texte est "
+"plus petit que la taille maximum. La valeur maximale de longueur est "
+"utilisée comme longueur fixe."
+
+#: Settings.ui.h:33
+msgid "Display running indicators on unfocused applications"
+msgstr "Afficher des indicateurs sur les applications an arrière-plan"
+
+#: Settings.ui.h:34
+msgid "Use the favorite icons as application launchers"
+msgstr "Utiliser les applications favorites comme lanceurs"
+
+#: Settings.ui.h:35
 msgid "Preview timeout on icon leave (ms)"
 msgstr "Délai de l'aperçu lorsqu'on quitte l'icône (ms)"
 
-#: Settings.ui.h:26
+#: Settings.ui.h:36
 msgid ""
 "If set too low, the window preview of running applications may seem to close "
 "too quickly when trying to enter the popup. If set too high, the preview may "
 "linger too long when moving to an adjacent icon."
 msgstr ""
-"Si trop bas, l'aperçu des fenêtres des applications actives peut "
-"sembler se fermer trop vite quand on essaie de cliquer sur l'aperçu. Si "
-"trop haut, l'aperçu peut rester trop longtemps quand on passe à une icône "
-"adjacente."
+"Si trop bas, l'aperçu des fenêtres des applications actives peut sembler se "
+"fermer trop vite quand on essaie de cliquer sur l'aperçu. Si trop haut, "
+"l'aperçu peut rester trop longtemps quand on passe à une icône adjacente."
 
-#: Settings.ui.h:27
+#: Settings.ui.h:37
+msgid "Time (ms) before showing (100 is default)"
+msgstr "Temps (ms) avant d'afficher (100 par défaut)"
+
+#: Settings.ui.h:38
+msgid "Enable window peeking"
+msgstr "Activer la vue de fenêtre"
+
+#: Settings.ui.h:39
+msgid ""
+"When hovering over a window preview for some time, the window gets "
+"distinguished."
+msgstr ""
+"Le survol sur la prévisualisation de la fenêtre, pendant un certain temps, "
+"la met en avant."
+
+#: Settings.ui.h:40
+msgid "Enter window peeking mode timeout (ms)"
+msgstr "Temps d'attente avant activation de la vue fenêtre"
+
+#: Settings.ui.h:41
+msgid ""
+"Time of inactivity while hovering over a window preview needed to enter the "
+"window peeking mode."
+msgstr ""
+"Temps d'inactivité pendant le survol de la prévisualisation d'une fenêtre "
+"nécessaire pour activer la vue de la fenêtre."
+
+#: Settings.ui.h:42
+msgid "Window peeking mode opacity"
+msgstr "Opacité de la vue de fenêtre"
+
+#: Settings.ui.h:43
+msgid ""
+"All windows except for the peeked one have their opacity set to the same "
+"value."
+msgstr ""
+"Toute fenêtre, mis à part celle prévisualisée, ont leur opacité mise à la "
```

Comment:
`Toute fenêtre` should maybe be `Toutes les fenêtres` if `ont` is used afterwards